collections/luci-ssl-nginx/root/etc/uci-defaults/70-luci

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+chmod +x /etc/init.d/luci
```

Comment:
Same here

Reply:
I am not sure, how to do it. We need a $(INSTALL_BIN) … somewhere, right? But where, if we want to use the Package/$(PKG_NAME)/install from luci.mk?

Reply:
Thats right. In LuCI you have to put the files into **root** with the right subdirectories and with the right mod flags and it will gets installed. See for example https://github.com/openwrt/luci/tree/master/applications/luci-app-adblock/root/etc/uci-defaults

Reply:
Oh, now I have a bunch of commits with the wrong username/email. Will continue tomorrow and change the mods of the files, too.